### DISCUSSION TITLE
Retrieve data for multiple disk for a single VM ?

### DISCUSSION THREAD

**Discussion by samos667** [proxmoxer/proxmoxer#129] [Q&A] [answered]
Is possible to retrieve data for multiple disk for a single VM ?

When I get data from instance.nodes("<node_name>").get("qemu") he return only infos from 1st disk.

**Comment by jhollowe** [comment, marked as answer]
It looks like the VM summary only contains information for the boot/root disk. To get information on other disks, you would want to look at the [nodes/{node}/qemu/{vmid}/config](https://pve.proxmox.com/pve-docs/api-viewer/index.html#/nodes/{node}/qemu/{vmid}/config) path.
